lib/users.test.js

```diff
@@ -96,32 +102,28 @@ describe('Sign up', () => {
       username: 'Batman',
       email: '1234@gmail.com',
       password: 'password',
-      update: jest.fn()
     }
     return expect(signUp(obj)).rejects.toThrow(
       'this account already exists'
     )
   })
+
   it('should create a user account', async () => {
+    const dataValues = await genUserInfoWithPw('abcd1234')
     mockFindOne.mockReturnValue(undefined)
     mockCreateAccount.mockReturnValue({
-      id: 5,
-      update: jest.fn()
+      dataValues: dataValues,
+      update: () => {}
     })
-
     const obj = {
-      username: 'Batman',
-      email: '1234@gmail.com',
-      password: 'password',
+      username: 'testuser',
+      password: 'abcd1234',
+      email: 'test.user@databases.com'
     }
-
-    await signUp(obj)
-
-    expect(mockCreateAccount.mock.calls.length).toEqual(1)
-    expect(mockCreateAccount.mock.calls[0][0].username).toEqual('Batman')
```

Comment:
so you decided to only check input and output of the function? You don't want to check if the `createAccount` is getting called?

Reply:
This way makes sure creatAccount is called